aspnetcore/tutorials/grpc/grpc-start.md

```diff
@@ -3,15 +3,269 @@ title: Create a .NET Core gRPC client and server in ASP.NET Core
 author: juntaoluo
 description: This tutorial shows how to create a gRPC Service and gRPC client on ASP.NET Core. Learn how to create a gRPC Service project, edit a proto file, and add a duplex streaming call.
 ms.author: johluo
-ms.date: 08/20/2021
+ms.date: 09/13/2021
 no-loc: [Home, Privacy, Kestrel, appsettings.json, "ASP.NET Core Identity", cookie, Cookie, Blazor, "Blazor Server", "Blazor WebAssembly", "Identity", "Let's Encrypt", Razor, SignalR]
 uid: tutorials/grpc/grpc-start
 ---
 # Tutorial: Create a gRPC client and server in ASP.NET Core
 
 By [John Luo](https://github.com/juntaoluo)
 
-::: moniker range=">= aspnetcore-5.0"
+::: moniker range=">= aspnetcore-6.0"
+This tutorial shows how to create a .NET Core [gRPC](xref:grpc/index) client and an ASP.NET Core gRPC Server. At the end, you'll have a gRPC client that communicates with the gRPC Greeter service.
+
+In this tutorial, you:
+
+> [!div class="checklist"]
+> * Create a gRPC Server.
+> * Create a gRPC client.
+> * Test the gRPC client service with the gRPC Greeter service.
+
+## Prerequisites
+
+# [Visual Studio](#tab/visual-studio)
+
+[!INCLUDE[](~/includes/net-prereqs-vs-6.0.md)]
+
+# [Visual Studio Code](#tab/visual-studio-code)
+
+[!INCLUDE[](~/includes/net-prereqs-vsc-6.0.md)]
+
+---
+
+## Create a gRPC service
+
+# [Visual Studio](#tab/visual-studio)
+
+* Start Visual Studio 2022 and select **Create a new project**.
+* In the **Create a new project** dialog, search for `gRPC`. Select **ASP.NET Core gRPC Service** and select **Next**.
+* In the **Configure your new project** dialog, enter `GrpcGreeter` for **Project name**. It's important to name the project *GrpcGreeter* so the namespaces match when you copy and paste code.
+* Select **Next**.
+* In the **Additional information** dialog, select **.NET 6.0 (Preview)** and then select **Create**.
+
+# [Visual Studio Code](#tab/visual-studio-code)
+
+The tutorial assumes familiarity with VS Code. For more information, see [Getting started with VS Code](https://code.visualstudio.com/docs)
+
+* Open the [integrated terminal](https://code.visualstudio.com/docs/editor/integrated-terminal).
+* Change to the directory (`cd`) that will contain the project.
+* Run the following commands:
+
+  ```dotnetcli
+  dotnet new grpc -o GrpcGreeter
+  code -r GrpcGreeter
+  ```
+
+  * The `dotnet new` command creates a new gRPC service in the *GrpcGreeter* folder.
+  * The `code` command opens the *GrpcGreeter* folder in a new instance of Visual Studio Code.
+
+---
+
+### Run the service
+
+  [!INCLUDE[](~/includes/run-the-app.md)]
+
+The logs show the service listening on `https://localhost:7042`.
```

Comment:
FYI the port is now random.

Reply:
Thanks, I will update the include and topic to indicate that!